app/__init__.py

```diff
@@ -1,110 +1,121 @@
 from flask import Flask, jsonify, request
+from psycopg2.extras import RealDictCursor
 import uuid
+from flask_jwt_extended import (
+    JWTManager, jwt_required, create_access_token,
+    get_jwt_identity
+)
+from .model import User, UserRequest, get_all_requests
+from db_connect import TrackerDB
 
 app = Flask(__name__)
-
-requests = []
-users = []
-session = {}
+app.config['JWT_SECRET_KEY'] = 'super-secret'
```

Comment:
Is this your secret key? it should be secret (as the name suggests 😆 ) so put in in your env file and source it from there.

Reply:
The key is secret now, for sure. Included it in an env file which is gitignored